spec/models/doctor_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Doctor, type: :model do
+  subject do
+    described_class.new(
+      name: 'Andrew Okoh',
+      email: 'a.okoh@gmail.com',
+      phonenumber: '08123467820',
+      specialty: 'Gynecologist',
+      fee: 200,
+      picture: 'https://img.freepik.com/free-photo/happy-black-male-doctor-using-tablet-computer-technology-medicine-concept_1262-12344.jpg?w=740&t=st=1671022983~exp=1671023583~hmac=5b3ed441b0fd6349225fa8ea682887b9ccddbd38c83877283adffffd75317083',
+      location: 'Abuja',
+      years_of_experience: 6
+    )
+  end
+
+  it 'is not valid without a name' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without an email' do
+    subject.email = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a location' do
+    subject.location = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a specialty' do
+    subject.specialty = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a picture' do
+    subject.picture = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without a phone number' do
+    subject.phonenumber = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'is not valid without years_of_experience' do
+    subject.years_of_experience = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
[OPTIONAL] Strongly consider adding some positive case scenarios. It's important to cover both positive and negative scenarios in testing. 
Not just `not valid` tests but `valid` tests too. 
Why:
[OPTIONAL]
Should always have positive and negative tests to ensure it works the way you expect it. Eg. What do you expect the output to be when it's not valid or when valid etc.
Consider making all your tests more extensive, not just this one. For this simple application, it may not catch a possible crippling bug, but in bigger applications, it definitely can. Consider this for all projects moving forward.

Reply:
@mcrd25  I will add positive tests as well as advised to increase the scope of the coverage.